specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/networkVirtualAppliance.json

```diff
@@ -915,6 +915,20 @@
             "$ref": "#/definitions/VirtualApplianceNicProperties"
           }
         },
+        "networkProfile": {
+          "type": "object",
+          "readOnly": false,
+          "description": "Network Profile containing configurations for Public and Private NIC",
+          "properties": {
+            "networkInterfaceConfigurations": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/NetworkVirtualApplianceNetworkInterfaceConfiguration"
+              }
+            }
+          },
+          "x-ms-identifiers": []
```

Comment:
`x-ms-identifiers` should be defined in `networkInterfaceConfigurations` block?

Reply:
but `x-ms-identifiers` should be used for `array` type while `networkProfile` is an `object` type. Could you double confirm?
https://azure.github.io/autorest/extensions/#x-ms-identifiers

Reply:
Yes you are right, I overlooked the usage in JSONs, I have corrected it.